daisy_workflows/image_build/debian/build.py

```diff
@@ -123,6 +124,50 @@ def main():
   synopsis_dest = os.path.join(outs_path, 'synopsis.json')
   utils.UploadFile('/tmp/synopsis.json', synopsis_dest)
 
+  # Create and upload metadata of the image and packages
+  logging.info('Creating image metadata.')
+  image = {
+      "family": "debian-9",
+      "name": re.match("debian-9-[\\w-]*", image_dest),
```

Comment:
we can't know the image name at the time of source creation, as this one source could be associated with many images. think this will have to come from gce api

Reply:
i thought this build.py is only invoked by debian9.wf.json workflow. so should I remove this field and populate later from gce api to image?

Reply:
yes, the 'name' should be the name from the gce image resource. the build workflow only produces the binary source for an image, the `root.tar.gz` file. it has no name, only family and destination path.